pyccel/parser/semantic.py

```diff
@@ -915,8 +915,16 @@ def _handle_function(self, func, args, **settings):
 
             return expr
```

Comment:
Shouldn't we call this variable `new_expr` as done below?

Reply:
Fixed in 008e068